simulate_robot_index: 2
angle_weight: 1.5
path_weight: 2.5
robot_weight: 1
